scania-digital-design-system/sdds
# fix(footer): moving theme rules to component folder

**Describe pull-request**  
moving theme rules to the component folder for the footer component

**Solving issue**  
Fixes: [AB#2217](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2217)

**How to test**  
1. Go to Storybook in production
2. Go to Storybook here for this PR 
3. Compare footer component - there should not be the difference
